CONTRIBUTING.md

```diff
@@ -1,28 +1,62 @@
 # Contributing
 
-## Development
+## Find or file an issue to work on
+
+If you're looking to add a feature or fix a bug and there's no issue filed yet,
+it's good to
+[file an issue](https://github.com/maplibre/maplibre-compose/issues/new/choose)
+first to have a discussion about the change before you start working on it.
+
+If you're new and looking for things to contribute, see our
+[good first issue](https://github.com/maplibre/maplibre-compose/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22)
+label. These issues are usually ready to work on and don't require deep
+knowledge of the library's internals.
+
+If you have particular knowledge of MapLibre, Android, iOS, or anything else
+relevant, see the
+[help wanted](https://github.com/maplibre/maplibre-compose/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22help%20wanted%22)
+label. These are issues that need input or guidance from folks with deeper
+expertise on some topic.
+
+## Set up your development environment
 
 Create a `local.properties` in the root of the project with paths to inform
 Gradle where to find the Android SDK:
 
 ```properties
-# Replace the paths with the actual paths on your machine
+# Replace the path with the actual path on your machine
 sdk.dir=/Users/username/Library/Android/sdk
 ```
 
-Follow
-[Jetbrains's instructions](https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-setup.html#get-help)
-for setting up an environment for KMP development.
+As there's no stable LSP for Kotlin Multiplatform, you'll want to use either
```

Comment:
What's an LSP?

Reply:
https://en.m.wikipedia.org/wiki/Language_Server_Protocol

Used by editors like vscode, neovim, etc to provide language intelligence like an IDE.

Reply:
Ah